src/ErrorBoundary.tsx

```diff
@@ -1,49 +1,78 @@
-import { ErrorBoundary as SentryErrorBoundary } from '@sentry/react';
-import { FallbackRender } from '@sentry/react/dist/errorboundary';
 import * as React from 'react';
+import { GenericReactErrorBoundaryError } from './error';
 import { ErrorBoundaryContext } from './ErrorBoundaryProvider';
 import { FallbackComponent } from './FallbackComponent';
-import { ErrorBoundaryProps, FallbackProps } from './types';
+import { ErrorBoundaryProps, ErrorBoundaryState } from './types';
 
-const wrapFallback = (Component: React.ElementType<FallbackProps>): FallbackRender => {
-  return props => (
-    <Component error={props.error} componentStack={props.componentStack} tryRecovering={props.resetError} />
-  );
-};
+export class ErrorBoundary<P extends object = {}> extends React.PureComponent<
```

Comment:
CLASS COMPONENTS??? That brings me back. You know they don't even teach those in code bootcamps anymore. 🤣

Reply:
well, there's no error hook 😆 if you need a react boundary, you're supposed to use a class component

Reply:
![22B](https://github.com/stoplightio/react-error-boundary/assets/587740/f778fdd6-a725-4303-bac4-6035ee144cd9)